antispam_bee.php

```diff
@@ -1101,13 +1101,13 @@ public static function precheck_incoming_request()
 
 		$post_id = (int) self::get_key( $_POST, 'comment_post_ID' );
 		// Form fields
-		$hidden_field = self::get_key($_POST, 'comment');
-		$plugin_field = self::get_key( $_POST, self::get_secret_for_post( $post_id ) );
+		$hidden_field = self::get_key( $_POST, 'comment' );
+		$plugin_field = self::get_key( $_POST, self::get_secret_name_for_post( $post_id ) );
 
 		// Hidden field check
 		if ( empty($hidden_field) && ! empty($plugin_field) ) {
 			$_POST['comment'] = $plugin_field;
-			unset( $_POST[ self::get_secret_for_post( $post_id ) ] );
+			unset( $_POST[ self::get_secret_name_for_post( $post_id ) ] );
```

Comment:
We could unset this too. But I think the general idea here is more get rid of the extra `$_POST`-field after the data has been transfered to `$_POST['comment']`